src/main/resources/application.conf

```diff
@@ -229,6 +229,7 @@ cpmeta{
 					{ label: "sitesspectral", format: "https://meta.fieldsites.se/ontologies/sites/spectralCsv"},
 					{ label: "sitescsv", format: "https://meta.fieldsites.se/ontologies/sites/simpleSitesCsv"},
 					{ label: "dailysitescsv", format: "https://meta.fieldsites.se/ontologies/sites/dailySitesCsv"},
+					{ label: "sitesImage", format: "https://meta.fieldsites.se/ontologies/sites/image"},
```

Comment:
InstanceServer ids have been in small letters so far. It's a minor point, of course, but could be good to maintain consistency.